Source/WTF/wtf/PlatformUse.h

```diff
@@ -397,3 +397,10 @@
     && (PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 140000)
 #define USE_NSSPELLCHECKER_GRAMMAR_CHECKING_POLICY 1
 #endif
+
+#if (PLATFORM(MAC) && __MAC_OS_X_VERSION_MAX_ALLOWED < 140000) \
+    || (PLATFORM(IOS_FAMILY) && __IPHONE_OS_VERSION_MAX_ALLOWED < 170000) \
+    || (PLATFORM(APPLETV) && __TV_OS_VERSION_MAX_ALLOWED < 170000) \
+    || (PLATFORM(WATCHOS) && __WATCH_OS_VERSION_MAX_ALLOWED < 100000)
+#define USE_CORE_TEXT_OPTICAL_SIZING_RANGE_CLAMPING_WORKAROUND 1
+#endif
```

Comment:
I'm not sure if we prefer having one guard for each flag, but this uses the same guard as `USE_VISION_CPU_ONLY_PROPERTY 1` and `USE_METAL_ARGUMENT_ACCESS_ENUMS`.

Reply:
Isn't the problem solved by rdar://108877507 being solved by CoreText?